package.json

```diff
@@ -270,6 +270,7 @@
   "devDependencies": {
     "@babel/core": "7.16.0",
     "@dump247/storybook-state": "1.6.1",
+    "@glif/filecoin-wallet-provider": "2.0.0-alpha.9",
```

Comment:
i think this should be a dependency, not a devDependency

Reply:
done